app/preprints/-components/submit/submission-flow/template.hbs

```diff
@@ -15,11 +15,13 @@ as |manager|>
             </div>
         </layout.heading>
         {{#if (is-mobile)}}
-            <layout.top>
+            <layout.top as |top|>
                 <div local-class='top-container'>
                     <div local-class='top-left'>
                         <Preprints::-Components::Submit::PreprintStateMachine::StatusFlow
                             @manager={{manager}}
+                            @leftNavToggle={{top.toggleSidenav}}
+                            @isMobile={{true}}
```

Comment:
Can remove `@isMobile` in this file too

Reply:
Ahh, so the `isMobile` argument isn't necessarily telling the component whether it's mobile or not, it's to say if the `status-flow-display` component should show/style a certain way